assets/eip-5218/ic3license/ic3license.tex

```diff
@@ -49,13 +59,14 @@ \section{Introduction}
 
 The \iccclicense is a copyright license specifically designed for use with NFTs. It links a creative work to an NFT so that when the NFT is transferred, so is the license. It is intended to be compatible with the NFT licensing standard defined in \eiplicense. 
 
-The \iccclicense has three optional features that a licensor can use or not when choosing a license:
+The \iccclicense has four optional features that a licensor can use or not when choosing a license:
 \begin{itemize}
-\item The license can be \commercial or \noncommercial. A \commercial license allows the user to make money from merchandise and other uses of the work; a \noncommercial license does not.
-\item The license can be \derivative or \noderivative. A \derivative license allows the user to make derivative works and adaptations, like remixes and video series; a \noderivative license does not.
-\item The license can be \ledger or \legal. A \ledger license means that the current state of the blockchain ledger is always authoritative for who owns the NFT and has rights under the license, even if the NFT is stolen or transferred by mistake. A \legal license gives the courts flexibility to correct ownership of the license in clear cases of theft and fraud.
+\item The license can be \commercial (C) or \noncommercial (NC). A \commercial license allows the user to make money from merchandise and other uses of the work; a \noncommercial license does not.
+\item The license can be \derivative (D), \derivativetracking (DT), \sharealike (SA), or \noderivative (ND). A \noderivative license allows the user to make derivative works and adaptations (like remixes and videos) without restriction. A \derivativetracking license allows derivative works but requires them to be registered and tracked as NFTs themselves. A \sharealike license is like \derivativetracking but also requires that the derivative works be relicensed under the same license. And a \noderivative license prohibits derivative works entirely. (These options are ordered from least restrictive to most restrictive.)
+\item The license can be \publiclicense (PL) or \nopubliclicense (NPL). A \publiclicense license, in addition to the specific rights granted to the user, grants to the public a broad copyright license to reproduce the work (but not to make derivative works). A \nopubliclicense license grants only the specific rights to the user.
+\item The license can be \ledger (Ledger) or \legal (Legal). A \ledger license means that the current state of the blockchain ledger is always authoritative for who owns the NFT and has rights under the license, even if the NFT is stolen or transferred by mistake. A \legal license gives the courts flexibility to correct ownership of the license in clear cases of theft and fraud.
```

Comment:
[Peer feedback] Consider the `legal` type to require specifying a jurisdiction to which court has power to correct ownership, confiscate or destroy?  
e.g. One option is  to https://eips.ethereum.org/EIPS/eip-5485

Reply:
@grimmelm: "One challenge here is that it's not always clear that any court at all has the power to force a transfer, destruction, etc. Another is that locking down a forum can make the license less portable because a purchaser might not be located in a convenient jurisdiction and could get into a dispute with someone else who is also far away from the chosen forum."
For the goal of a minimal license for NFTs in general, we try to avoid these issues. We recommend people of interest add terms on jurisdiction after carefully working through the consequences.